src/routes/app-router/AppRouter.jsx

```diff
@@ -6,22 +6,22 @@ import { MainLayout } from '../../layouts/main-layout/MainLayout';
 import { AdminPage } from '../../pages/admin/AdminPage';
 
 import { HomePage } from '../../pages/user-pages/home/HomePage';
-import { Page1 } from '../../pages/user-pages/page1/Page1';
+import { TeamPage } from '../../pages/user-pages/TeamPage/TeamPage';
```

Comment:
Now here is a good example of naming react component
It should starts with capital letter